### PR TITLE
Fix req

### PR DESCRIPTION
**Pull request by taliaga**
@c0c0n3 I'll leave it for you to:
- Upgrade requests (see security warning at home)
- Update Docker image to use pipenv file

:)

### CODE REVIEW

**Comment by c0c0n3** [comment]
Besides upgrading deps and migrating to `pipenv`, I've done a bunch of adjustments/slight improvements to make our code work with the upgraded libs.

File: Dockerfile [hunk 1]
```diff
@@ -2,8 +2,11 @@ FROM python:3.6
 
 RUN mkdir -p /src/ngsi-timeseries-api
 
-COPY requirements.txt /src/ngsi-timeseries-api/requirements.txt
+COPY Pipfile /src/ngsi-timeseries-api/Pipfile
+COPY Pipfile.lock /src/ngsi-timeseries-api/Pipfile.lock
 
+RUN pip install pipenv
```
Comment (taliaga) [inline Dockerfile]: Let's move this line above the two copy lines and then feel free to merge!

Comment (c0c0n3) [reply Dockerfile]: done!

**Comment by taliaga** [comment]
Thanks!

**Pull request merged**